src/main/java/frc/trigon/robot/utilities/Conversions.java

```diff
@@ -0,0 +1,16 @@
+package frc.trigon.robot.utilities;
+
+public class Conversions {
+    public static double hundredMsToSec(double ms)
+    {
+        return ms*10;
+    }
+    public static double hundredMsToMin(double ms)
+    {
+        return hundredMsToSec(ms)*60;
+    }
+    public static double hundredMsToTickPerMin(double ms)
+    {
+        return hundredMsToMin(ms)*2048;
+    }
+}
```

Comment:
add conversion to RPM its easiest to work in RPM because we know the max of the falcons is around 6000 RPM.